src/program.hpp

```diff
@@ -340,6 +462,7 @@ struct cvk_program : public _cl_program, api_object {
     spir_binary m_binary{SPV_ENV_VULKAN_1_0};
     std::vector<cvk_sampler_holder> m_literal_samplers;
     std::vector<VkPushConstantRange> m_push_constant_ranges;
+    std::unordered_map<std::string, cvk_entry_point*> m_entry_points;
```

Comment:
I'm wondering whether we should use `std::unique_ptr` here. I don't think matters too much (beyond shortening the destructor's code).

Reply:
Sure, will do.